### ISSUE TITLE
Deprecation warning for `jax.experimental.optimizers` and `jax.experimental.stax`

### ISSUE THREAD

**Issue by corneliusroemer** [pyro-ppl/numpyro#1267]
When importing `numpyro` I get a `jax` deprecation warning for `jax.experimental.optimizers` and `jax.experimental.stax`

```python
import datetime as dt

import arviz as az
import matplotlib.pyplot as plt
import matplotlib.ticker as ticker
import numpy as np
import numpyro
import numpyro.distributions as dist
import pandas as pd
import seaborn as sns
from jax import random
from numpyro.handlers import reparam
from numpyro.infer import MCMC, NUTS, Predictive
from numpyro.infer.reparam import LocScaleReparam
from scipy.special import expit, logit
from sklearn.preprocessing import LabelEncoder

assert numpyro.__version__.startswith("0.8.0")

/opt/hostedtoolcache/Python/3.10.1/x64/lib/python3.10/site-packages/jax/experimental/optimizers.py:28: 
FutureWarning: jax.experimental.optimizers is deprecated, import jax.example_libraries.optimizers instead
  warnings.warn('jax.experimental.optimizers is deprecated, '

/opt/hostedtoolcache/Python/3.10.1/x64/lib/python3.10/site-packages/jax/experimental/stax.py:28: 
FutureWarning: jax.experimental.stax is deprecated, import jax.example_libraries.stax instead
  warnings.warn('jax.experimental.stax is deprecated, '
```

Am I doing the imports wrong or should `numpyro` change some what is imported from `jax` in the next release?

**Comment by MarcoGorelli** [comment]
Hey @corneliusroemer - this was fixed in https://github.com/pyro-ppl/numpyro/pull/1221, the next release should avoid the warning

**Comment by corneliusroemer** [comment]
Great, couldn't find an issue mentioning this yet.

Shall I close?

**Comment by MarcoGorelli** [comment]
Yeah, am not a maintainer but am fairly confident this is OK to close 😄

**Comment by corneliusroemer** [comment]
Resolved in #1221
